tgui/packages/tgui/interfaces/NTRecruiter.js

```diff
@@ -0,0 +1,201 @@
+import { useBackend } from '../backend';
+import { Box, Button, LabeledList, Section, Flex } from '../components';
+import { LabeledListItem } from '../components/LabeledList';
+import { Window } from '../layouts';
+
+export const NTRecruiter = (props, context) => {
+  const { act, data } = useBackend(context);
+  const {
+    gamestatus,
+    cand_name,
+    cand_gender,
+	  cand_age,
+	  cand_species,
+	  cand_planet,
+	  cand_job,
+	  cand_records,
+	  cand_curriculum,
+    reason,
+  } = data;
+  if (gamestatus === 0) {
+  return (
+    <Window resizable>
+      <Window.Content>
+        <Section py="140px">
+          <Flex direction="column">
+            <Flex.Item fontSize="31px" color="white" pb="10px" textAlign="center" bold>
+              Nanotrasen Recruiter Simulator
+            </Flex.Item>
+            <Flex.Item fontSize="16px" textAlign="center" color="label">
+              Work as the Nanotrasen recruiter and avoid hiring incompetent employees!
+            </Flex.Item>
+          </Flex>
+          <Button
+            mt="50px"
+            textAlign="center"
+            lineHeight={2}
+            fluid
+            icon="play"
+            color="green"
+            content="Begin Game"
+            onClick={() => act('start_game')}
+          />
+          <Button
+            mt="10px"
+            textAlign="center"
+            lineHeight={2}
+            fluid
+            icon="info"
+            color="blue"
+            content="Guide"
+            onClick={() => act('instructions')}
+          />
+        </Section>
+      </Window.Content>
+    </Window>
+    )}
+  if (gamestatus === 1) {
+  return (
+    <Window resizable>
+      <Window.Content>
+        <Section
+          color="grey"
+          title="Guide"
+          buttons={
+            <Button
+              icon="arrow-left"
+              content="Main Menu"
+              onClick={() => act('back_to_menu')}
+            />
+          }
+        >
+        <LabeledList>
+          <LabeledListItem label="1#" color="silver">
+            To win this game you must hire/dismiss <b>FIVE</b> candidates,
+            one choice made wrong leads to game over.
+          </LabeledListItem>
+          <LabeledListItem label="2#" color="silver">
+            Make the right choice by truly putting yourself into
+            the skin of a recruiter working for Nanotrasen!
+          </LabeledListItem>
+          <LabeledListItem label="3#" color="silver">
+            <b>Unique</b> characters may appear, pay attention to them!
+          </LabeledListItem>
+          <LabeledListItem label="4#" color="silver">
+            Make sure to pay attention to details like ages,
+            planet names, jobs offered by NT and even the species of the candidate!
```

Comment:
```suggestion
            Make sure to pay attention to details like age,
            planet names, the requested job and even the species of the candidate!
```
Might be more clearly worded this way - up to you